src/main.go

```diff
@@ -70,11 +72,12 @@ func main() {
 	} else if len(os.Args) > 1 {
 		port = ":" + os.Args[1]
 	}
-	log.Infoln("initializing...")
+	log.Infoln("initializing bench routes...")
 	conf = parser.New(configurationFilePath)
 	conf.Load().Validate()
 	setDefaultServicesState(conf)
 	intervals := conf.Config.Interval
+	// Loads workers on to the memory
```

Comment:
```suggestion
	// Loads workers on to the memory.
```
I would rather say "Initialise workers" but please feel free to ignore this.